lib/std/os/linux.zig

```diff
@@ -4532,8 +4532,20 @@ pub const clockid_t = enum(u32) {
     BOOTTIME = 7,
     REALTIME_ALARM = 8,
     BOOTTIME_ALARM = 9,
-    SGI_CYCLE = 10,
-    TAI = 11,
+    // In the linux kernel header file (time.h) is the following note:
+    // * The driver implementing this got removed. The clock ID is kept as a
+    // * place holder. Do not reuse!
+    // Therefore, calling clock_gettime() with these IDs will result in an error.
+    // SGI_CYCLE = 10,
+    // TAI = 11,
+    _,
+};
+
+pub const CLOCK_ID = clock_id;
+
+pub const clock_id = enum(u32) {
```

Comment:
Can you add a short comment here and on clockid_t differentiating them?  (I think just labelling which API/syscall they're aimed at would be sufficient? clock_gettime vs timerfd_create?)

Reply:
yes, make sense.